exercises/practice/roman-numerals/.approaches/introduction.md

```diff
@@ -0,0 +1,204 @@
+# Introduction
+
+There is no single, obvious solution to this exercise, but a diverse array of working solutions have been used.
+
+## General guidance
+
+Roman numerals are limited to positive integers from 1 to 3999 (MMMCMXCIX).
+In the version used for this exercise, the longest string needed to represent a Roman numeral is 14 characters (MMDCCCLXXXVIII).
+Minor variants of the system have been used which represent 4 as IIII rather than IV, allowing for longer strings, but those are not relevant here.
+
+The system is inherently decimal: the number of human fingers has not changed since ancient Rome, nor the habit of using them for counting. However, there is no zero value available, so Roman numerals represent powers of 10 with different letters (I, X, C, M), not by position (1, 10, 100, 1000).
```

Comment:
```suggestion
The system is inherently decimal: the number of human fingers has not changed since ancient Rome, nor the habit of using them for counting.
However, there is no zero value available, so Roman numerals represent powers of 10 with different letters (I, X, C, M), not by position (1, 10, 100, 1000).
```